notebooks/explore-molecular-generative-models-latent-spaces.ipynb

```diff
@@ -12,7 +12,7 @@
   },
```

Comment:
Same in this notebook, the traces are huge but the source code seems to be identical

Reply:
Here the images were missing for some reason.